Sources/Mantis/CropView/CropMaskViewManager.swift

```diff
@@ -5,9 +5,7 @@
 //  Created by Echo on 10/28/18.
 //  Copyright © 2018 Echo. All rights reserved.
 //
-
 import UIKit
-
```

Comment:
I don't think it is necessary  to remove these lines.

Reply:
yes,I will keep them